src/partisan_hiscamp_membership_strategy.erl

```diff
@@ -102,29 +105,28 @@ get_centroid_two_clusters(#hiScamp{membership=Membership0, heap=Heap, level1=L1,
         true ->
             % can group nodes as theyre the closest within the same threshold
         %% NODE CONNECTIONS
+            % join(Min1, Min2)
             ok;
         false -> 
             heaps:merge(heaps:from_list(sets:to_list(L2), sets:to_list(L1)))
     end.
 
-% hierarchial_clustering(#hiScamp{membership=Memberhsip0, heap=Heap, level1=L1, level2=L2}=State0) ->
-  %  ok.
-
-leave(#hiScamp{membership=Membership0, level1=L1, level2=L2}=State0, Node) ->
-    case partisan_config:get(tracing, ?TRACING) of 
-        true ->
-            lager:info("~p: Issuing remove_subscription for node ~p.", [node(), Node]);
-        false ->
-            ok
-    end,
-    Membership = sets:del_element(Node, Membership0),
-    MembershipList0 = membership_list(State0),
+%change to leave_levels?
+ %leave(#hiScamp{membership=Membership0, level1=L1, level2=L2}=State0, Node) ->
+  %  case partisan_config:get(tracing, ?TRACING) of 
+   %     true ->
+    %        lager:info("~p: Issuing remove_subscription for node ~p.", [node(), Node]);
+     %   false ->
+      %      ok
+%    end,
+ %   Membership = sets:del_element(Node, Membership0),
+  %  MembershipList0 = membership_list(State0),
 %% Gossip
-    Message = {remove_subscription, Node},
-    OutgoingMessages = lists:map(fun(Peer) -> {Peer, {protocol, Message}} end, MembershipList0),
-    State = State0#hiScamp{membership=Membership},
-    MembershipList = membership_list(State),
-    {ok, MembershipList, OutgoingMessages, State}.
+%    Message = {remove_subscription, Node},
+ %   OutgoingMessages = lists:map(fun(Peer) -> {Peer, {protocol, Message}} end, MembershipList0),
+  %  State = State0#hiScamp{membership=Membership},
+   % MembershipList = membership_list(State),
+   % {ok, MembershipList, OutgoingMessages, State}.
```

Comment:
This change isn't going to work.  These functions *have* to have the return value format of the 4-tuple with membership, messages and state.